pkg/controllers/vdb/http_server.go

```diff
@@ -41,9 +39,3 @@ func hasCompatibleVersionForHTTPServer(vrec *VerticaDBReconciler, vdb *vapi.Vert
 	}
 	return true
 }
-
-// genHTTPServerCtrlQuery returns a query used to interact
-// with the http server.
-func genHTTPServerCtrlQuery(action string) string {
```

Comment:
What do we need `httpServerMode ` for?

Reply:
Excellent question. I knew I had more to do when I opened this task. I added a commit that will deprecate that parameter. We now setup the TLS certs for nma solely based on the deployment method (vclusterops only). I had to move the e2e-http-server leg so its only run in vcluster mode. The leg was rename to e2e-leg-6 too.